spec/unit/provider/service/upstart_spec.rb

```diff
@@ -34,6 +34,43 @@ def lists_processes_as(output)
     expect(described_class.default?).to be_truthy
   end
 
+  context "upstart daemon existence confine" do
+    # We have a separate method here because our search for the upstart daemon
+    # confine expects it to be the last confine declared in the upstart provider.
```

Comment:
Is this expectation expressly stated anywhere else? I cant find it in the actual provider code...?

Reply:
Can you clarify what you mean by expectation? Not sure I follow.

Reply:
"... because our search for the upstart daemon confine expects it to be the last confine declared ..."

Reply:
Oh. That one's from looking at the `upstart` provider and seeing that the confine block added in this PR is the last one that's declared.

Reply:
Cool. Good to go. Want me to merge?

Reply:
Please do. Thanks :).